tests/functional/managementAgent/managementProcess.js

```diff
@@ -0,0 +1,368 @@
+    OrbitSimulatorEvent,
+} = require('./orbitServerSimulator');
+const {
+    ChannelMessageV0,
+    MessageType,
+} = require('../../../lib/management/ChannelMessageV0');
+
+
+const PAYLOAD_MSG_TYPE = MessageType.CHANNEL_PAYLOAD_MESSAGE;
+const CONFIG_OVERLAY_MSG_TYPE = MessageType.CONFIG_OVERLAY_MESSAGE;
+const METRICS_REQ_MSG_TYPE = MessageType.METRICS_REQUEST_MESSAGE;
+const METRICS_REPORT_MSG_TYPE = MessageType.METRICS_REPORT_MESSAGE;
+const CHAN_CLOSE_MSG_TYPE = MessageType.CHANNEL_CLOSE_MESSAGE;
+
+
+/* Test the management process.
+ *
+ * This process connects to Orbit WebSocket server and is itself a WebSocket
+ * server. Testing it requires:
+ * - an orbit simulator,
+ * - a WebSocket client to connect to it,
+ * - HTTP server to answer the stats requests,
+ * - a socket server for payload messages.
+ **/
+describe('Management process', function testSuite() {
+    this.timeout(120000);
+
+    /* Make sure the process management send payload message to a local host
+     * socket to be able to receive this message in this test.  */
+    assert.strictEqual(process.env.SECURE_CHANNEL_DEFAULT_FORWARD_TO_HOST,
+                       'localhost');
+
+    function createWs(path) {
+        const host = _config.managementAgent.host;
+        const port = _config.managementAgent.port;
+        return new WebSocket(`ws://${host}:${port}/${path || 'watch'}`);
+    }
+
+    it('should not listen on others routes than `watch`', done => {
+        const ws = createWs('wrong_path');
+        const msg = 'management agent process should not listen this route';
+
+        ws.on('open', () => { done(new Error(msg)); });
+        ws.on('unexpected-response', (_, response) => {
+            assert.strictEqual(response.statusCode, 400);
+
+            done();
+        });
+    });
+
+    it('should listen on `watch` route', done => {
+        const ws = createWs();
+
+        ws.on('open', done);
+        ws.on('error', error => { done(error); });
+    });
+
+    it('should terminate the connection when a client does not answer ping',
+       done => {
+           this.timeout(2 * CHECK_BROKEN_CONNECTIONS_FREQUENCY_MS);
+
+           const ws = createWs();
+
+           ws.on('close', (code, reason) => {
+               assert.strictEqual(code, WS_STATUS_IDDLE.code);
+               assert.strictEqual(reason, WS_STATUS_IDDLE.reason);
+
+               done();
+           });
+
+           ws.on('error', error => { done(error); });
+
+           ws.on('message', () => {
+               /* Ugly eventTarget internal fields hacking to avoid this web
+                * socket to answer to ping messages. It will make
+                * the management agent to close the connection after a timeout.
+                * Defining an onPing event does not help, this internal
+                * function is still called. */
+               ws._receiver._events.ping = function noop() {};
+           });
+       });
+
+    describe('with orbit simulator', () => {
```

Comment:
not usefull anymore to have a describe specific to tests with orbit. It was done because I wanted to use before/after Each hooks but finally did not, so let's get an indentation level back.